rental.rb

```diff
@@ -15,4 +15,13 @@ def initialize(date, person, book)
   def to_s
     "Date: #{@date}, Book \"#{book.title}\" by #{book.author}"
   end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'date' => @date,
+      'book_title' => @book.title,
+      'person_id' => @person.id
+    }.to_json(*args)
+  end
```

Comment:
- [ ] [OPTIONAL] Sorry that it wasn't pointed out earlier and therefore this is an OPTIONAL suggestion, however when creating some teachers, students, books and rentals the application works how it is supposed to however when closing the application I get an error and I'm not able to restart it anymore. see below the print screen of the error.
![Przechwytywanie](https://user-images.githubusercontent.com/39240820/142230648-5644bce2-184c-4963-ab8a-ecb7c88a21e3.PNG)